pyogrio/tests/test_geopandas_io.py

```diff
@@ -12,6 +12,8 @@
 try:
     import geopandas as gp
     from geopandas.testing import assert_geodataframe_equal
+    import pygeos as pg
```

Comment:
```suggestion
    import pygeos
```

(preference to keep the pygeos import simple without alias, following pygeos' docs)

Reply:
Also, can you move this import inside the function with a `pygeos = pytest.importorskip("pygeos")` (that's also how it is done elsewehere, pygeos is not a strictly required dependency; although we should maybe make it a required testing dependency ..)